### ISSUE TITLE
OptionsPlugins cannot be opened a second time after accepting or cancelling

### ISSUE THREAD

**Issue by omaraa** [imagej/imagej#163]
To reproduce:
1. Open the OptionsLookAndFeel plugin (Edit >> Options >> Look and Feel...)
2. Press Cancel
3. Open the OptionsLookAndFeel plugin a second time
 - Plugin does not open and IJ status bar shows message "Command cancelled: Look and Feel..."

This issue affects all plugins extending OptionsPlugin.

**Comment by imagejan** [comment]
This is a scijava-commons issue and likely due to the `cancelReason` not being reset on subsequent runs of a `org.scijava.command.DynamicCommand`, see [these lines](https://github.com/scijava/scijava-common/blob/608831075023f9ddf053691a82c2be24458f337c/src/main/java/org/scijava/command/DynamicCommand.java#L125-L140).

**Comment by ctrueden** [comment]
Thanks for the report, @omaraa. I confirm this bug occurs with the latest ImageJ, 2.0.0-rc-59.

**Comment by ctrueden** [comment]
@imagejan Right, because [`OptionsPlugin` extends `DynamicCommand` and implements `SingletonPlugin`](https://github.com/scijava/scijava-common/blob/scijava-common-2.62.1/src/main/java/org/scijava/options/OptionsPlugin.java#L74-L75)—i.e.: there is only one instance of each `OptionsPlugin` per context. But one of the preconditions of commands is that they only run once per instance lifetime. So we need to make an architectural decision here about the best way to proceed. As things stand, options plugins are a dirty hack.
